send-data/secure-syslog.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Send data from syslog to Axiom over a secure connection'
+description: 'This page explains how to send data securely from a syslog logging system to Axiom.'
+overview: 'Standard protocol for message logging'
+sidebarTitle: Secure syslog
+tags:
+  ['syslog', 'axiom documentation', 'documentation', 'axiom', 'guide', 'syslog logs', 'tcp', 'udp', 'docker']
+logoId: 'syslog'
+darkLogoId: 'syslogLight'
+---
+
+import Prerequisites from "/snippets/standard-prerequisites.mdx"
+import Replacement from "/snippets/standard-replacement.mdx"
+import SetUpEndpoint from "/snippets/set-up-endpoint.mdx"
+
+The Secure Syslog endpoint allows you to send syslog data to Axiom over a secure connection. With the Secure Syslog endpoint, the logs you send to Axiom are encrypted using SSL/TLS.
+
+## Syslog limitations and recommended alternatives
+
+Syslog is an outdated protocol from the 1980s. Some of the limitations are the following:
+
+- Lack of error reporting and feedback mechanisms when issues occur.
+- Inability to gracefully terminate the connection. This can result in missing data.
+
+<Note>
+For a more reliable and modern logging experience, consider using tools like [Vector](https://vector.dev/) to receive syslog messages and [forward them to Axiom](/send-data/vector). This approach bypasses many of syslog’s limitations.
+</Note>
+
+<Prerequisites />
+
+<SetUpEndpoint endpointName="Secure Syslog" />
+
+## Configure syslog client
+
+Other requirements:
+
+- **Message size limit:** Axiom currently enforces a 64KB per-message size limit. This is in line with RFC5425 guidelines. Any message exceeding the limit causes the connection to close because Axiom doesn’t support ingesting truncated messages.
+- **TLS requirement:** Axiom only supports syslog over TLS, specifically following RFC5425. Configure your syslog client with a self-generated certificate or a custom server certificate. For more information, see the [RFC Series documentation](https://www.rfc-editor.org/rfc/rfc5425).
+- **Port requirements:** TCP log messages are sent on TCP port `601`.
```

Comment:
```suggestion
- **Port requirements:** TCP log messages are sent on TCP port `6514`.
```
(unless platform tell you something different), but that's the normal TLS syslog port